### PR TITLE
Use SLCORE through RPC

### PR DESCRIPTION
**Pull request by henryju**

### CODE REVIEW

File: org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/backend/SonarLintEclipseHeadlessRpcClient.java [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+    return false;
+  }
+
+  @Override
+  public void didReceiveServerHotspotEvent(DidReceiveServerHotspotEvent params) {
+    // No Hotspots in SLE
+  }
+
+  @Override
+  public void didUpdatePlugins(String connectionId) {
```
Comment (thahnen) [inline org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/backend/SonarLintEclipseHeadlessRpcClient.java:145]: Isn't it something similar to what we had prior in the `NodeJsManager` implementation?
> SonarLintCorePlugin.getConnectionManager().getConnections().forEach(f -> f.stop());

Comment (henryju) [reply org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/backend/SonarLintEclipseHeadlessRpcClient.java:145]: Is it a question for me? :)

Comment (thahnen) [reply org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/backend/SonarLintEclipseHeadlessRpcClient.java:145]: It was more for myself for later, but maybe you have an answer already 😄

Comment (henryju) [reply org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/backend/SonarLintEclipseHeadlessRpcClient.java:145]: Yes, I think stopping all the connected engines is the way to go.

**Pull request closed without merging**